pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -505,6 +505,8 @@ const (
 	ClusterV1Beta1GeneratorName             = "cluster/v1beta1"
 	PodDisruptionBudgetV1GeneratorName      = "poddisruptionbudget/v1beta1"
 	PodDisruptionBudgetV2GeneratorName      = "poddisruptionbudget/v1beta1/v2"
+	DaemonsetV1Beta1GeneratorName           = "extensions/v1beta1"
+	DaemonsetV1Beta2GeneratorName           = "apps/v1beta2"
```

Comment:
Change `DaemonsetV1Beta2GeneratorName` to `DaemonsetAppsV1beta2GeneratorName` to make it more explicit?

Reply:
👍 agree, will updated. thank you